packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -221,20 +222,20 @@ $ms-Persona-presenceSizeXl: 28px;
 }
 
 .ms-Persona-details {
+  box-sizing: border-box;
   padding: 0 12px;
   vertical-align: middle;
   @include text-align(left);
   @include padding-left($ms-Persona-sizeMd + $ms-Persona-imageDetailsLgSpace);
   display: table-cell;
+  width: 100%;
```

Comment:
This is unexpected behavior for sure. That said I don't think that this is necessarily the correct way to fix it. This almost seems like it should be flex or float so that it stays left aligned more.